README.md

```diff
@@ -72,8 +72,9 @@ If you are using [Expo](https://expo.io/), you also need to add this to `app.jso
 ```json
 {
   "expo": {
-    "packagerOpts": {
-      "config": "metro.config.js"
+    "packagerOpts": { 
+      "config": "metro.config.js", 
+      "sourceExts": ["js", "jsx", "svg", "svgx"] 
```

Comment:
The `.svgx` extension is only a fallback extension for older React Native versions, so it should not be included here as people should be using the standard `.svg` extension for the images.

Reply:
Thank you for this PR @chin8628 
I added this 
```
"sourceExts": ["js", "jsx", "svg"]
```
and it's working for expo `"sdkVersion": "33.0.0",`

Reply:
@kristerkari I've removed 'svgx' extension in a new commit.